src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1498,11 +1498,18 @@ public static Tombstone TableValueToTombstone(JsonOperationContext context, ref
                 else
                 {
                     flags = localFlags | documentFlags;
+                    var revisionsStorage = DocumentDatabase.DocumentsStorage.RevisionsStorage;
 
+                    if (nonPersistentFlags.Contain(NonPersistentDocumentFlags.FromReplication) &&
+                        revisionsStorage.Configuration == null &&
```

Comment:
If we have a local tombstone and we receive a tombstone which is a conflict resolution.. wouldn't we strip the flag here?